### ISSUE TITLE
zoom

### ISSUE THREAD

**Issue by 0323pin** [woelper/oculante#40]
Would it be possible to add an alternative to zoom _in_ (`+`) and _out_ (`-`) with these keys?
Mouse wheel is not very useful when there's no mouse 😄

**Comment by woelper** [comment]
Sure. That's a good idea. I'll see if I can get them to zoom at a good rate and play with increments.

**Comment by 0323pin** [comment]
Cheers and thanks from the _tiling wm users_

**Comment by woelper** [comment]
After some thought, this is also an accessibility issue. I guess people without mouse would also like to pan the image horizontally and vertically?

**Comment by 0323pin** [comment]
Yes 👍

**Comment by 0323pin** [comment]
Thanks!
I've just merged 0.6.3, tomorrow I'll build and merge 0.6.5
Too tired to do it now.

Btw, have you thought about keeping a ChangeLog?
Here's my Log Message for the update from 0.6.1 to 0.6.3,

```
graphics/oculante: update to 0.6.3

0.6.3
- fix missing vsync argument on windows

0.6.2
- draw brush preview
- update to latest notan
- fix test
- add undocumented key shortcut
- clean-up
```

**Comment by woelper** [comment]
Yes, I have thought about it, but I am too lazy to manually write one. I'll look into creating one from commits and be a bit fancier with their wording I guess...
